### PR TITLE
fix: make it compatible with playwright-test

### PR DESCRIPTION
**Pull request by mxschmitt**
Gets released soon: https://playwright.dev/docs/next/test-intro/

I tested it locally, seems to work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,21 @@ To activate it in your Jest environment you have to include it in your configura
...
 
-### With Playwright test runner
+### With [Playwright test runner](https://playwright.dev/docs/test-intro)
 
 To activate with the Playwright test runner, use `expect.extend` to add the `expect-playwright` matchers.
 
 ```js
-// folio.config.ts
-import { expect } from "@playwright/test"
+// foo.spec.ts
```
[the remark this answers is not in the document]
Comment (mxschmitt) [reply README.md]: Good catch. Guess who wrote the docs for that! :smile: 

updated it.

File: README.md [hunk 2]
```diff
@@ -22,16 +22,33 @@ To activate it in your Jest environment you have to include it in your configura
...
+
+// ...
+```
+
+Inside the tests you can add the import to have TypeScript support.
+
+```js
+// foo.spec.ts
+import { test, expect } from "@playwright/test"
+import "expect-playwright"
```
Comment (mskelton) [inline README.md:44]: Wondering if we need this section given the TypeScript section at the bottom of the readme:

https://github.com/playwright-community/expect-playwright#typescript

**Review by mskelton — APPROVED** [review]
Good stuff!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mskelton) [reply README.md]: Nice :)

**Pull request merged**